client/lib/url/index.js

```diff
@@ -26,8 +26,24 @@ function isHttps( url ) {
 	return url && startsWith( url, 'https://' );
 }
 
+const urlWithoutHttpRegex = /^https?:\/\//;
+
+/**
+ * Returns the supplied URL without the initial http(s).
+ * @param  {String}  url The URL to remove http(s) from
+ * @return {?String}     URL without the initial http(s)
+ */
+function withoutHttp( url ) {
+	if ( ! url ) {
```

Comment:
Empty strings will be falsey. It may be safer to return `url` in this case if you look at where this is being called.